server/lib/src/idm/ldap.rs

```diff
@@ -643,15 +642,54 @@ mod tests {
         let pce = UnixPasswordChangeEvent::new_internal(UUID_ADMIN, TEST_PASSWORD);
 
         assert!(idms_prox_write.set_unix_account_password(&pce).is_ok());
-        assert!(idms_prox_write.commit().is_ok());
+        assert!(idms_prox_write.commit().is_ok()); // Committing all configs
+
+        // default UNIX_PW bind (default is set to true)
+        // Hence allows all unix binds
+        let admin_t = ldaps
+            .do_bind(idms, "admin", TEST_PASSWORD)
+            .await
+            .unwrap()
+            .unwrap();
+        assert!(admin_t.effective_session == LdapSession::UnixBind(UUID_ADMIN));
+        let admin_t = ldaps
+            .do_bind(idms, "admin@example.com", TEST_PASSWORD)
+            .await
+            .unwrap()
+            .unwrap();
+        assert!(admin_t.effective_session == LdapSession::UnixBind(UUID_ADMIN));
 
+        // Setting UNIX_PW_BIND flag to false:
+        // Hence all of the below tests will fail (asserts are still satisfied)
```

Comment:
Maybe reword to "authentication will fail" rather than the test failing. The test is succeeding at showing auth failing!